core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/SnapshotApiServiceSpec.scala

```diff
@@ -283,7 +640,9 @@ class SnapshotApiServiceSpec extends ApiServiceSpec {
     Get(s"${testData.workspaceSubmittedSubmission.path}/snapshots?offset=0&limit=10") ~>
       sealRoute(services.snapshotRoutes) ~>
       check {
-        assertResult(StatusCodes.ImATeapot) {response.status}
+        assertResult(StatusCodes.ImATeapot) {
```

Comment:
Why do we have an `ImATeapot` response…?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/418

You'll have to ask AppServices for specifics on this particular test I'm afraid

Reply:
to distinguish from a generic error such as 500, which could be thrown/wrapped by many layers … using an uncommon code like 418 ImATeapot helps the test know that it's bubbling up exactly the error that was thrown by the `SnapshotApiServiceSpecWorkspaceManagerDAO` mock. Plus … it's cute.